sdk/ml/azure-ai-ml/tests/internal/unittests/test_component.py

```diff
@@ -41,6 +41,15 @@ def test_validate_internal_component(self):
         validation_result = validate_component(yaml_path)
         assert validation_result.passed, repr(validation_result)
 
+    def test_component_inputs_with_bool_and_date_value(self):
+        yaml_path = r"tests/test_configs/internal/command-component/command-linux/" \
+                    r"component_with_bool_and_data_input/component.yaml"
+        component = load_component(yaml_path)
+        assert component.inputs["bool_input"].default == "true"
```

Comment:
what if input value is "True" instead of "true"?

Reply:
Why there is no recording change?

Reply:
Because this PR only changed ut?

Reply:
Because in smile the founded default value of inputs is true/false in lowercase, and confirmed with Clement that it is converted to lowercase

Reply:
I checked the component under "tests/test_configs/internal", and there is no component that contains this kind of scene

Reply:
May need to add test case on that